src/aiokeycloak/types/__init__.py

```diff
@@ -0,0 +1,24 @@
+import access_token
```

Comment:
What kind of import?

Reply:
I has modify kind of import access_token `3eea470`